app/views/account_dashboard_statuses.scala.html

```diff
@@ -0,0 +1,127 @@
+pager = cases,
+containerClass = Some(containerClass),
+onChange = page => controllers.routes.IndexController.getApplicationsAndRulings(page)
+)
+}
+
+@main_template(
+title = messages("index.title"),
+appConfig = appConfig,
+contentClass = Some("column-full"),
+bodyClasses = None) {
+
+
+<h1 class="heading-xlarge">
+ <span class="heading-secondary">@messages("application.dashboard.status.title")</span>
+ @messages("index.heading")
+</h1>
+
+@components.button_link(messages("site.start"), routes.BeforeYouStartController.onPageLoad.url, id = Some("start-application"))
+
+<div id="applications-rulings-list">
+ @paginate("mb-2", "top")
+ <table id="applications-rulings-list-table">
+  <thead>
+  <tr>
+   <th scope="col">@messages("case.tabs.applications.table.reference")</th>
+   <th scope="col">@messages("case.tabs.applications.table.itemName")</th>
+   <th scope="col">@messages("case.tabs.applications.table.dateSubmitted")</th>
+   <th scope="col">@messages("case.tabs.applications.table.status")</th>
+   <th scope="col"><span class="visuallyhidden">@messages("case.application.ruling.table.download")</span></th>
+  </tr>
+  </thead>
+  <tbody id="applications-rulings-list-body">
+  @for((c: Case, index: Int) <- cases.results.zipWithIndex) {
+  <tr id="applications-rulings-list-row-@{index}">
+   <td id="applications-rulings-list-row-@{index}-reference">@c.reference</td>
+   <td id="applications-rulings-list-row-@{index}-item_name">@c.application.goodName</td>
+   <td id="applications-rulings-list-row-@{index}-date_submitted">@Dates.format(c.createdDate)</td>
+   <td id="applications-rulings-list-row-@{index}-status">
+    @c.status match {
+    case CaseStatus.NEW => {
+     <span class="govuk-tag govuk-tag--blue">@messages("case.application.status.submitted")</span>
+    }
+    case CaseStatus.OPEN | CaseStatus.SUSPENDED => {
+     <span class="govuk-tag govuk-tag--blue">@messages("case.application.status.inProgress")</span>
+    }
+    case CaseStatus.REFERRED => {
+     <span class="govuk-tag govuk-tag--yellow">@messages("case.application.status.infoRequested")</span>
+    }
+    case CaseStatus.SUPPRESSED | CaseStatus.REJECTED => {
+     <span class="govuk-tag govuk-tag--red"> @messages("case.application.status.rejected")</span>
+    }
+    case CaseStatus.COMPLETED if c.hasActiveDecision => {
+    @if(c.daysDifference.get > 120){
+     <span class="govuk-tag govuk-tag--green"> @messages("case.application.status.approvedRuling")</span>
+    } else {
+     <span class="govuk-tag govuk-tag--red">@messages("case.application.status.approvedRulingExpiring", c.daysDifference.get)</span>
+    }
+    }
+    case CaseStatus.COMPLETED if c.hasExpiredDecision => {
+     <span class="govuk-tag govuk-tag--grey">@messages("case.application.ruling.status.expired")</span>
+    }
+    case CaseStatus.CANCELLED => {
+     <span class="govuk-tag govuk-tag--grey">@messages("case.application.status.cancelled")</span>
+    }
+    case _ => {}
+    }
+   </td>
```

Comment:
Maybe we should have a helper function or a separate component to figure out the tag to display?